.github/workflows/publish-nargo.yml

```diff
@@ -198,3 +198,14 @@ jobs:
           make_latest: false
           overwrite: true
           tag: ${{ format('{0}-{1}', 'nightly', steps.date.outputs.date) }}
+
+  dispatch-publish-es:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dispatch to publish-es-packages
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: publish-es-packages.yml
+          ref: master
+          token: ${{ secrets.NOIR_REPO_TOKEN }}
+          inputs: "{ \"noir-ref\": \"${{ inputs.tag || env.GITHUB_REF }}\", \"nightly\": ${{ github.event_name == 'schedule' }} }"
```

Comment:
Why do we have this here rather than adding a schedule trigger to `public-es-packages.yml`?

Reply:
I don't know how trustful towards scheduler we can be, the doc says:
> Scheduled workflows run on the latest commit on the default or base branch. 

but do not clarify if one can rely on two workflows scheduled at the same time would have same commit. Therefore there would be chance of drift between nargo and js packages (if PR's wedge between).

This is still correct though?

Reply:
Hmm, that's a good point. Perhaps we can have a separate workflow which just has a scheduled trigger and dispatches to both of these workflows. Not a blocker though.

Reply:
Actually, let's make this change. We currently check that we can run builds of nargo on each pull request so we're currently trying and failing to make a JS release on each PR.